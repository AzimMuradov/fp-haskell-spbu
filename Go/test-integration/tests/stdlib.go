func main() {
  print(1, " ", "2", 3, " ", 4);                    // should print "1 23 4" (without '\n')
  println(2, [4] string {"x", "y", "z", "."}, nil); // should print "2 [x y z .] nil"
  println(len("alpha"));                            // should print "5"
  println(len([19] int {}));                        // should print "19"
  panic("scary error");                             // should panic with "panic: scary error"
}
