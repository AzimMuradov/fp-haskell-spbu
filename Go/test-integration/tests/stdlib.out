1 23 42 [x y z .] nil
5
19
panic: scary error
